inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/logical.cpp

```diff
@@ -78,19 +81,19 @@ INSTANTIATE_TEST_CASE_P(smoke_EltwiseLogicalInt,
                                 ::testing::Values(ngraph::helpers::InputLayerType::PARAMETER),
                                 ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
                                 ::testing::Values(CommonTestUtils::DEVICE_MYRIAD),
-                                ::testing::Values(Config{{InferenceEngine::MYRIAD_DETECT_NETWORK_BATCH, CONFIG_VALUE(NO)}})),
+                                ::testing::Values(additional_config)),
                         LogicalLayerTest::getTestCaseName);
 
 INSTANTIATE_TEST_CASE_P(smoke_EltwiseLogicalNotInt,
-                        LogicalLayerTestVPU,
+                        LogicalLayerTest,
                         ::testing::Combine(
                                 ::testing::ValuesIn(LogicalLayerTest::combineShapes(inputShapesNot)),
-                                ::testing::Values(InferenceEngine::Precision::BOOL),
+                                ::testing::Values(InferenceEngine::Precision::I32),
```

Comment:
Does it related to missing BOOL input and output precision support in CNNNetwork?

Reply:
I'm not sure about CNNNetwork, but Myriad plugin does not support `bool`